src/components/appData/EventForm.jsx

```diff
@@ -1,73 +1,130 @@
 import React from "react";
 import PropTypes from "prop-types";
-import { Button, TextField, MenuItem, Select } from "@mui/material";
+import {
+  Box,
+  Button,
+  InputLabel,
+  FormControl,
+  TextField,
+  MenuItem,
+  Select,
+  Typography,
+  Avatar,
+  OutlinedInput
+} from "@mui/material";
+import { Image } from "@mui/icons-material";
 
 const EventForm = ({ value, onSubmitForm }) => {
+  //reusable form to add new event or edit existing event
   let dateValue = "";
-  console.log("data from form", value);
+  //the MongoDB saves date in UTC time zone. Here the converter to current data for rendering it on date field and saving it in MongoDB
   if (value && value.date !== null) {
-    console.log(new Date(value.date).getUTCDate());
     let day = new Date(value.date).getUTCDate();
     let month = new Date(value.date).getUTCMonth();
```

Comment:
You need to add 1 to the month because the getUTCMonth returns the month starting with 0.
Ex:
```
let month = new Date(value.date).getUTCMonth() + 1;
```

Reply:
Oh no! I do not know how I missed this part! Thank you!